src/applications/disability-benefits/all-claims/components/ComboBox.jsx

```diff
@@ -99,9 +203,19 @@ export class ComboBox extends React.Component {
       filteredOptions: [],
       highlightedIndex: 0,
     });
-    this.inputRef.current.focus();
     const { onChange } = this.props;
     onChange(option);
+    // Send focus to input element for additional user input.
+    this.inputRef.current.focus();
```

Comment:
When the user selects an option from the dropdown with the mouse, the focus should move back to the input field. When i was testing this locally, the focus was not moving back to the input field from the list. I played around with the below to set focus and it seemed to work but just wanted to share. 
```
const vaTextFieldShadowRoot = this.inputRef.current.shadowRoot
const inputField = vaTextFieldShadowRoot.querySelector('input')
inputField.focus()
```